templates/docs/examples/patterns/navigation/_script.js

```diff
@@ -37,8 +37,13 @@ function initNavDropdowns(containerClass) {
     toggle.addEventListener('click', function (e) {
       e.preventDefault();
 
-      closeAllDropdowns(toggles);
-      toggleDropdown(toggle, true);
+      const wasOpen = toggle.parentNode.classList.contains('is-active');
+      if (wasOpen) {
+        closeAllDropdowns(toggles);
+      } else {
+        closeAllDropdowns(toggles.filter((item) => item !== toggle));
+        toggleDropdown(toggle, true);
```

Comment:
Currently, if click-target was not open, we close all others and open it.

The alternative you mention is to always close all and open the click-target if it was not open. That is how I had it initially committed. I understood your comment above, that this alternative is redundant. Because it closes the click-target and then opens it again.

Now I am a bit confused. Do we want to have this redundancy in close-and-immediately-reopen, or do we want to avoid it?